gcloud/logging/test_client.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest2
+
+
+class TestClient(unittest2.TestCase):
+
+    def _getTargetClass(self):
+        from gcloud.logging.client import Client
+        return Client
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
+    def test_ctor(self):
+        PROJECT = 'PROJECT'
+        CREDS = _Credentials()
+        CLIENT_OBJ = self._makeOne(project=PROJECT, credentials=CREDS)
```

Comment:
Not really any assertable state yet.  This module is a placeholder (as is `test_connection`, after c5f19ab),  b83c3df adds assertions what little is there.